Script starting...
Waiting for Jupyter Lab server to open port 15597...
TIMING - Starting wait at: Fri Aug 23 07:27:33 AM EDT 2024
++ date
+ echo 'TIMING - Starting main script at: Fri Aug 23 07:27:33 AM EDT 2024'
TIMING - Starting main script at: Fri Aug 23 07:27:33 AM EDT 2024
+ '[' -n '' ']'
+ '[' -z /tmp ']'
+ '[' /tmp == /tmp ']'
++ mktemp -d
+ TMPDIR=/tmp/tmp.zRihpa0QF0
+ export TMPDIR
+ cd /hpc/home/ajj54
+ export GROUPDIR=/hpc/group/borsuklab
+ GROUPDIR=/hpc/group/borsuklab
+ echo 'GROUPDIR is /hpc/group/borsuklab'
GROUPDIR is /hpc/group/borsuklab
+ BPath=/tmp/tmp.zRihpa0QF0:/tmp
+ BPath+=,/work:/work
+ BPath+=,/hpc/group/borsuklab:/hpc/group/borsuklab
+ BPath+=,/opt/apps/models:/opt/apps/models
+ echo /tmp/tmp.zRihpa0QF0:/tmp,/work:/work,/hpc/group/borsuklab:/hpc/group/borsuklab,/opt/apps/models:/opt/apps/models
/tmp/tmp.zRihpa0QF0:/tmp,/work:/work,/hpc/group/borsuklab:/hpc/group/borsuklab,/opt/apps/models:/opt/apps/models
++ date
+ echo 'TIMING - Starting jupyter at: Fri Aug 23 07:27:33 AM EDT 2024'
TIMING - Starting jupyter at: Fri Aug 23 07:27:33 AM EDT 2024
+ '[' no == no ']'
+ singularity -vvv exec --bind=/tmp/tmp.zRihpa0QF0:/tmp,/work:/work,/hpc/group/borsuklab:/hpc/group/borsuklab,/opt/apps/models:/opt/apps/models /opt/apps/containers/community/codeplus2023/codeplus-2023.sif jupyter lab --config=/hpc/home/ajj54/ondemand/data/sys/dashboard/batch_connect/sys/bc_jupyter_codeplus2023/output/df2a7996-555d-4a1e-8bac-7ffc2d33bb7f/config.py
VERBOSE: Not forwarding APPTAINER_VERBOSE environment variable
VERBOSE: Setting HOME=/hpc/home/ajj54
VERBOSE: Setting PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
VERBOSE: Set messagelevel to: 4
VERBOSE: Starter initialization
VERBOSE: Check if we are running as setuid: 1
VERBOSE: Drop root privileges
VERBOSE: Drop root privileges permanently
VERBOSE: Spawn stage 1
VERBOSE: Execute stage 1
VERBOSE: stage 1 exited with status 0
VERBOSE: Get root privileges
VERBOSE: Change filesystem uid to 1546375
VERBOSE: Spawn master process
VERBOSE: Create mount namespace
VERBOSE: Entering in mount namespace
VERBOSE: Create mount namespace
VERBOSE: Spawn RPC server
VERBOSE: Execute master process
VERBOSE: Default mount: /proc:/proc
VERBOSE: Default mount: /sys:/sys
VERBOSE: Default mount: /dev:/dev
VERBOSE: Found 'bind path' = /etc/localtime, /etc/localtime
VERBOSE: Found 'bind path' = /etc/hosts, /etc/hosts
VERBOSE: Default mount: /tmp:/tmp
VERBOSE: Default mount: /var/tmp:/var/tmp
VERBOSE: Default mount: /etc/resolv.conf:/etc/resolv.conf
VERBOSE: Serve RPC requests
VERBOSE: Checking for template passwd file: /var/apptainer/mnt/session/rootfs/etc/passwd
VERBOSE: Creating passwd content
VERBOSE: Creating template passwd file and appending user data: /var/apptainer/mnt/session/rootfs/etc/passwd
VERBOSE: Default mount: /etc/passwd:/etc/passwd
VERBOSE: Checking for template group file: /var/apptainer/mnt/session/rootfs/etc/group
VERBOSE: Creating group content
VERBOSE: Default mount: /etc/group:/etc/group
VERBOSE: /hpc/home/ajj54 found within container
VERBOSE: rpc server exited with status 0
VERBOSE: Execute stage 2
[W 2024-08-23 07:27:53.718 ServerApp] jupyter_server_terminals | error adding extension (enabled: True): Jupyter Server Terminals requires Jupyter Server 2.0+
    Traceback (most recent call last):
      File "/opt/conda/envs/rapids/lib/python3.10/site-packages/jupyter_server/extension/manager.py", line 320, in add_extension
        extpkg = ExtensionPackage(name=extension_name, enabled=enabled)
      File "/opt/conda/envs/rapids/lib/python3.10/site-packages/jupyter_server/extension/manager.py", line 166, in __init__
        super().__init__(*args, **kwargs)
      File "/opt/conda/envs/rapids/lib/python3.10/site-packages/traitlets/traitlets.py", line 1361, in __init__
        value = self._traits[key]._cross_validate(self, getattr(self, key))
      File "/opt/conda/envs/rapids/lib/python3.10/site-packages/traitlets/traitlets.py", line 746, in _cross_validate
        value = obj._trait_validators[self.name](obj, proposal)
      File "/opt/conda/envs/rapids/lib/python3.10/site-packages/traitlets/traitlets.py", line 1233, in __call__
        return self.func(*args, **kwargs)
      File "/opt/conda/envs/rapids/lib/python3.10/site-packages/jupyter_server/extension/manager.py", line 175, in _validate_name
        self._module, self._metadata = get_metadata(name)
      File "/opt/conda/envs/rapids/lib/python3.10/site-packages/jupyter_server/extension/utils.py", line 60, in get_metadata
        module = importlib.import_module(package_name)
      File "/opt/conda/envs/rapids/lib/python3.10/importlib/__init__.py", line 126, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
      File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "/opt/conda/envs/rapids/lib/python3.10/site-packages/jupyter_server_terminals/__init__.py", line 9, in <module>
        raise RuntimeError("Jupyter Server Terminals requires Jupyter Server 2.0+")
    RuntimeError: Jupyter Server Terminals requires Jupyter Server 2.0+
[I 2024-08-23 07:28:04.759 ServerApp] dask_labextension | extension was successfully linked.
[I 2024-08-23 07:28:04.869 ServerApp] jupyter_server_fileid | extension was successfully linked.
[I 2024-08-23 07:28:04.869 ServerApp] jupyter_server_proxy | extension was successfully linked.
[I 2024-08-23 07:28:04.877 ServerApp] jupyter_server_ydoc | extension was successfully linked.
[W 2024-08-23 07:28:04.880 LabApp] 'ip' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
[W 2024-08-23 07:28:04.881 LabApp] 'port' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
[W 2024-08-23 07:28:04.881 LabApp] 'port_retries' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
[W 2024-08-23 07:28:04.881 LabApp] 'password' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
[W 2024-08-23 07:28:04.881 LabApp] 'base_url' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
[W 2024-08-23 07:28:04.881 LabApp] 'base_url' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
[W 2024-08-23 07:28:04.881 LabApp] 'allow_origin' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
[W 2024-08-23 07:28:04.881 LabApp] 'notebook_dir' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
[W 2024-08-23 07:28:04.881 LabApp] 'disable_check_xsrf' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
[W 2024-08-23 07:28:04.885 ServerApp] notebook_dir is deprecated, use root_dir
[I 2024-08-23 07:28:04.885 ServerApp] jupyterlab | extension was successfully linked.
[I 2024-08-23 07:28:04.892 ServerApp] nbclassic | extension was successfully linked.
[I 2024-08-23 07:28:04.929 ServerApp] notebook_shim | extension was successfully linked.
[I 2024-08-23 07:28:04.930 ServerApp] panel.io.jupyter_server_extension | extension was successfully linked.
[I 2024-08-23 07:28:04.930 ServerApp] voila.server_extension | extension was successfully linked.
[W 2024-08-23 07:28:04.963 ServerApp] WARNING: The Jupyter server is listening on all IP addresses and not using encryption. This is not recommended.
[I 2024-08-23 07:28:04.965 ServerApp] notebook_shim | extension was successfully loaded.
[I 2024-08-23 07:28:04.966 ServerApp] dask_labextension | extension was successfully loaded.
[I 2024-08-23 07:28:04.967 FileIdExtension] Configured File ID manager: ArbitraryFileIdManager
[I 2024-08-23 07:28:04.967 FileIdExtension] ArbitraryFileIdManager : Configured root dir: /hpc/home/ajj54
[I 2024-08-23 07:28:04.967 FileIdExtension] ArbitraryFileIdManager : Configured database path: /hpc/home/ajj54/.local/share/jupyter/file_id_manager.db
[I 2024-08-23 07:28:04.987 FileIdExtension] ArbitraryFileIdManager : Successfully connected to database file.
[I 2024-08-23 07:28:04.987 FileIdExtension] ArbitraryFileIdManager : Creating File ID tables and indices with journal_mode = DELETE
[I 2024-08-23 07:28:05.019 ServerApp] jupyter_server_fileid | extension was successfully loaded.
[I 2024-08-23 07:28:05.580 ServerApp] jupyter_server_proxy | extension was successfully loaded.
[I 2024-08-23 07:28:05.581 ServerApp] jupyter_server_ydoc | extension was successfully loaded.
[I 2024-08-23 07:28:05.582 LabApp] JupyterLab extension loaded from /opt/conda/envs/rapids/lib/python3.10/site-packages/jupyterlab
[I 2024-08-23 07:28:05.582 LabApp] JupyterLab application directory is /opt/conda/envs/rapids/share/jupyter/lab
[I 2024-08-23 07:28:05.587 ServerApp] jupyterlab | extension was successfully loaded.
[I 2024-08-23 07:28:05.598 ServerApp] nbclassic | extension was successfully loaded.
[I 2024-08-23 07:28:05.599 ServerApp] panel.io.jupyter_server_extension | extension was successfully loaded.
[I 2024-08-23 07:28:05.620 ServerApp] voila.server_extension | extension was successfully loaded.
[I 2024-08-23 07:28:05.622 ServerApp] Serving notebooks from local directory: /hpc/home/ajj54
[I 2024-08-23 07:28:05.622 ServerApp] Jupyter Server 1.23.6 is running at:
[I 2024-08-23 07:28:05.622 ServerApp] http://localhost:15597/node/dcc-core-04/15597/lab
[I 2024-08-23 07:28:05.622 ServerApp]  or http://127.0.0.1:15597/node/dcc-core-04/15597/lab
[I 2024-08-23 07:28:05.622 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[W 2024-08-23 07:28:05.651 ServerApp] No web browser found: could not locate runnable browser.
Discovered Jupyter Lab server listening on port 15597!
TIMING - Wait ended at: Fri Aug 23 07:28:05 AM EDT 2024
Generating connection YAML file...
[I 2024-08-23 07:28:19.856 ServerApp] 302 POST /node/dcc-core-04/15597/login (10.183.18.196) 3.07ms
[W 2024-08-23 07:28:22.652 ServerApp] 404 GET /node/dcc-core-04/15597/api/me?1724412501120 (10.183.18.196) 202.59ms referer=https://dcc-ondemand-01.oit.duke.edu/node/dcc-core-04/15597/lab?
[I 2024-08-23 07:28:25.582 ServerApp] New terminal with automatic name: 1
[I 2024-08-23 07:28:26.471 LabApp] Build is up to date
WARNING:bokeh.server.util:Host wildcard '*' will allow connections originating from multiple (or possibly all) hostnames or IPs. Use non-wildcard values to restrict access explicitly
[I 2024-08-23 07:28:31.241 ServerApp] EOF on FD 20; stopping reading
[I 2024-08-23 07:28:31.343 ServerApp] Terminal 1 closed
[I 2024-08-23 07:28:36.662 ServerApp] New terminal with automatic name: 1
slurmstepd: error: *** JOB 14948968 ON dcc-core-04 CANCELLED AT 2024-08-23T07:29:00 ***
